sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -174,6 +140,30 @@ class FileCatalog(Base):
     ingestion_date = Column(DateTime(timezone=True))
 
 
+class SPICEFiles(Base):
+    """SPICE files table."""
+
+    __tablename__ = "spice_files"
+
+    file_path = Column(String, nullable=False, primary_key=True, unique=True)
+    start_date = Column(DateTime, nullable=True)
+    end_date = Column(DateTime, nullable=True)
+    extension = Column(String, nullable=False)
```

Comment:
Yes, I added version to both SPICE and Ancillary now. The SPICE one I left as optional because I think there may be some SPICE files without versions like `de430.bsp` or the main ones that NAIF gives us.